app/templates/views/public/application_review.html

```diff
@@ -3,23 +3,23 @@
 <h2 style="text-align:center;">Gallery</h2>
 <div class ="row">
     <label for="name">Name:</label>
-        <p id="name">John Doe</p>
+        <p id="name">{{user.name}}</p>
```

Comment:
I changed user to form, since we really are not using data stored in the user table, but data stored in the form table